scripts/zones/Norg/Zone.lua

```diff
@@ -36,6 +38,14 @@ function onZoneIn(player, prevZone)
 end
 
 function onRegionEnter(player, region)
+    switch (region:GetRegionID()): caseof
+    {
+        [1] = function (x)  -- An Undying Pledge cs trigger
+            if player:getCharVar("anUndyingPledgeCS") == 1 then
+            player:startEvent(226)
```

Comment:
indentation needed.  i notice some of these files still have tabs in them.  look in your text editor for a setting that's something like "convert existing tabs into spaces", then open and save each file you touched with this PR.  that should convert all the tabs.

note that "convert existing tabs into spaces" is not the same thing as "convert new tabs into spaces" -- editor will probably have two different settings.  I'd turn both on for this project.  Also useful is the setting to "trim trailing whitespace at end of lines".